esmvalcore/preprocessor/_mask.py

```diff
@@ -226,6 +227,55 @@ def mask_landseaice(cube, fx_files, mask_out):
     return cube
 
 
+def mask_glaciated(cube, mask_out):
+    """
+    Mask out glaciated areas
+
+    It applies a Natural Earth mask. Note that for computational reasons
+    only the 10 largest polygons are used for masking.
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        data cube to be masked.
+
+    mask_out: str
+        "glaciated" to mask out glaciated areas
+
+    Returns
+    -------
+    iris.cube.Cube
+        Returns the masked iris cube.
+
+    Raises
+    ------
+    ValueError
+        Error raised if masking on irregular grids is attempted.
+        Irregular grids are not currently supported for masking
+        with Natural Earth shapefile masks.
+    """
+
+    # Dict to store the Natural Earth masks
+    cwd = os.path.dirname(__file__)
+    # read glaciated shapefile
+    shapefiles = {
+        'glaciated': os.path.join(cwd, 'ne_masks/ne_10m_glaciated_areas.shp'),
+    }
+    if mask_out == 'glaciated':
+        cube = _mask_with_shp(cube, shapefiles[mask_out],
+                              [1859, 1860, 1861, 1857, 1858,
+                               1716, 1587, 1662, 1578, 1606, ]
+                             )
```

Comment:
move the closing bracket right under `cube` so FLAKE shuts up

Reply:
There are conflicting complaints here apparently, I noticed this before. So when following your suggestion `pyLint` starts to complain. I will slightly rewrite the code to get around this.

This is the message from `pyLint`:
```
esmvalcore/preprocessor/_mask.py
Wrong continued indentation (remove 1 space).
Time to fix: 5 minutes 

        cube = _mask_with_shp(cube, shapefiles[mask_out],
                              [1859, 1860, 1861, 1857, 1858,
                               1716, 1587, 1662, 1578, 1606, ]
                              )
        logger.debug(
            "Applying glaciated areas mask from Natural Earth"
Why is this an issue?
bad-continuation (C0330)
Wrong %s indentation%s%s. TODO

Related code pattern
bad-continuation (C0330)
Wrong %s indentation%s%s. TODO

Python Code Style
Powered by PyLint
```

Reply:
Or maybe I moved the wrong bracket... Related to this,  the outcome of my local run of `prospector` does not always match the outcome from `circleCI`. Is this supposed to match one to one? How to achieve this?

Reply:
Bas, have a look now, I ran `yapf -i esmvalcore/preprocessor/_mask.py` which is the best tool to use when you not sure about formatting; this will align the formatting to Codacy standards

Reply:
*CircleCI

Reply:
Good one! I only don't like that it now spends many lines on `integers` through expanding the list over the lines... But well. Not a big deal, let's proceed :)